code/src/cljs/sixsq/nuvla/ui/edge_detail/views.cljs

```diff
@@ -557,107 +471,600 @@
             ])]]])))
 
 
-(defn SummarySection
+(defn TabOverview
   []
-  (let [nuvlabox  (subscribe [::subs/nuvlabox])
-        nb-status (subscribe [::subs/nuvlabox-status])
-        can-edit? (subscribe [::subs/can-edit?])
-        ssh-creds (subscribe [::subs/nuvlabox-ssh-keys])
-        acl-open  (r/atom false)
-        tr        (subscribe [::i18n-subs/tr])]
-
+  (let [nuvlabox    (subscribe [::subs/nuvlabox])
+        locale      (subscribe [::i18n-subs/locale])
+        nb-status   (subscribe [::subs/nuvlabox-status])
+        ssh-creds   (subscribe [::subs/nuvlabox-ssh-keys])
+        tr          (subscribe [::i18n-subs/tr])
+        edit        (r/atom false)
+        old-nb-name (r/atom (:name @nuvlabox))
+        close-fn    #(do
+                       (reset! edit false)
+                       (refresh (:id @nuvlabox)))]
     (fn []
-      (let [status       @(subscribe [::edge-subs/status-nuvlabox (:id @nuvlabox)])
+      (let [{:keys [id name description created updated
+                    version refresh-interval owner tags ssh-keys]} @nuvlabox
             {:keys [hostname ip docker-server-version
-                    operating-system architecture last-boot]} @nb-status
+                    operating-system architecture last-boot
+                    status nuvlabox-api-endpoint
+                    nuvlabox-engine-version docker-plugins] status-updated :updated} @nb-status
+            online-status  @(subscribe [::edge-subs/status-nuvlabox id])]
+        (when (not= (count ssh-keys) (count (:associated-ssh-keys @ssh-creds)))
+          (dispatch [::events/get-nuvlabox-ssh-keys ssh-keys]))
+        [ui/TabPane
+         [ui/Grid {:columns   2,
+                   :stackable true
+                   :padded    true}
+          [ui/GridRow
+           [ui/GridColumn {:stretched true}
+            [ui/Segment {:secondary   true
+                         :color       "blue"
+                         :raised      true}
+             [:h4 "NuvlaBox "
+              (when nuvlabox-engine-version
+                [ui/Label {:circular  true
+                           :color     "blue"
+                           :size      "tiny"}
+                 nuvlabox-engine-version])]
+             [ui/Table {:basic  "very"
+                        :padded false}
+              [ui/TableBody
+               [ui/TableRow
+                [ui/TableCell "ID"]
+                [ui/TableCell [values/as-link id :label id]]]
+               (when name
+                 [ui/TableRow
+                  [ui/TableCell "Name"]
+                  [ui/TableCell [ui/Icon {:name "pencil"
+                                          :on-click #(reset! edit true)
+                                          :style  {:cursor  "pointer"}}]
+                   " "
+                   (if @edit
+                     [ui/Input {:default-value name
```

Comment:
If acl doesn't allow edit this should not be proposed to the user as editable

Reply:
to be addressed in https://github.com/nuvla/ui/issues/483